docs/handbook/spending-money.md

```diff
@@ -0,0 +1,74 @@
+# Spending money
+
+There are many occassions when you will need to spend company money.
+
+PostHog is a lean organization - the less we spend, the more time we have to make sure the company takes off. However, it is more important you are productive, healthy and happy.
+
+Please just spend company money like it's your own.
+
+If it's a trivial expense, just buy it. We'll provide you with a company card with a $500/month spending limit for this reason.
+
+If it's a larger expense and doesn't fit into the items here, please **raise a policy suggestion for it as a pull request** in this doc, so we can document our decision making into our policy rather than making everything case by case.
+
+## Trivial expenses
+
+Just do it.
+
+This means expenses that are under $75 one off or under $20/month recurring that we can cancel easily.
+
+Make sure you keep copies for all receipts. We *cannot* pay you back for anything without a receipt.
+
+## Making larger purchases
+
+If your purchase fits within the policy below, there is no need to ask. We **cannot** pay you back for anything without a receipt.
+
+You may not have enough space on your company card if you're a new starter, just ask James (and if he's unavailable, Tim) to increase your limit.
+
+## Equipment
```

Comment:
Is there any flex regarding these expense suggestions? For example, I'm pretty satisfied with most of my setup (laptop, desks, chair, etc) currently and don't mind forgoing some of the equipment. But, something that might help in the future would be coverage on coworking/something along the same lines.

Reply:
Related to this, before the apocalypse, I used to work a lot in cafes. Not all the time, but at 3-4€ per coffee per 2h, it adds up. We invoiced those to the company. We used an app called [datamolino](https://www.datamolino.com/) to directly send those invoices to Xero, our accounting software.

What's your policy regarding this?

Reply:
#25 created, will write that up next